Userland/Libraries/LibLine/InternalFunctions.cpp

```diff
@@ -494,8 +494,10 @@ void Editor::case_change_word(Editor::CaseChangeOp change_op)
             m_buffer[m_cursor] = to_ascii_lowercase(m_buffer[m_cursor]);
         }
         ++m_cursor;
-        m_refresh_needed = true;
     }
+
+    m_refresh_needed = true;
+    m_chars_touched_in_the_middle = start;
```

Comment:
This is still broken! If `start` is `0` then it will not be refreshed.

Fixing now... (just gonna hard-code it to `1`)